service/src/main.rs

```diff
@@ -741,6 +751,31 @@ fn enclave_account<E: EnclaveBase>(enclave_api: &E) -> AccountId32 {
 
 // Alice plays the faucet and sends some funds to the account if balance is low
 fn ensure_account_has_funds(api: &mut Api<sr25519::Pair, WsRpcClient>, accountid: &AccountId32) {
+	let funding_amount = amount_to_be_funded(api, accountid);
+	if 0 < funding_amount {
+		bootstrap_funds_from_alice(api, accountid, funding_amount);
+	}
+}
+
+// Amount that needs to be funded for this account to have at least existential_deposit * MIN_FUND_INCREASE_FACTOR (1_000_000_000_000)
+fn amount_to_be_funded(api: &mut Api<sr25519::Pair, WsRpcClient>, accountid: &AccountId32) -> u128 {
+	// check account balance
+	let free = api.get_free_balance(accountid).unwrap();
```

Comment:
We should probably start using proper error handling instead of `unwrap()`? This function should return a `Result<u128>` instead of unwrapping